android/app/src/main/res/values/strings.xml

```diff
@@ -22,7 +22,7 @@
     <string name="menu_locations">Locations</string>
     <string name="menu_settings">Settings</string>
     <string name="menu_share">Share</string>
-    <string name="menu_about">About us</string>
+    <string name="menu_about">About Us</string>
```

Comment:
Please handle minor changes in different PRs

Your PR should have 1:1 correspondence with the issue it handles

Furthermore, these minor issues can have small PRs that will get merged instantly, and here they just make the review process difficult